Yoast/wordpress-seo
# Add accessible attributes to the SVG icons.

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

- adds back the attributes `role="img" aria-hidden="true" focusable="false"` to the SVGs that were removed in https://github.com/Yoast/wordpress-seo/commit/b66b588ff6927c84dbe4698f3aa1ada140d5208b#diff-160b3b8bd4badfd1cfc7300519d11d95
- for consistency, the attributes are added to _all_ the SVG files, not only to the SVGs that are meant to be used as icons, but also to the ones meant to be used as images

## Test instructions

These attributes don't affect in any way the visual appearance of the SVGs. 
- `role="img" aria-hidden="true"` ensures the SVGs are perceived as images and then not announced by assistive technologies: we hide them from assistive technologies because we're using them as icons or purely decorative images that don't need to be announced
- `focusable="false"`solves an IE11 bug, where SVGs become focusable when used inside a focusable element (e.g. a button): when this happens, users are forced to tab twice to navigate through a button 

A good place where to test the IE11 thing, is the Gutenberg toolbar:
 
<img width="374" alt="screen shot 2018-08-08 at 17 37 27" src="https://user-images.githubusercontent.com/1682452/43847907-0f20a9d8-9b32-11e8-854a-81f29520c67a.png">

- navigate the toolbar using the keyboard and notice:
  - before the patch, you need to tab twice on the Yoast button
  - after the patch, you need to tab only once, as expected

Worth nothing all this was already tested in #8140 / #9276 and this PR just restores the previous state.

For the future, I'd suggest to consider to have a standardized process to generate and update the SVG icons.

Fixes #10591
